app/src/test/java/com/example/myapplication/CredentialsManagerTest.kt

```diff
@@ -21,20 +21,19 @@ class CredentialsManagerTest {
     @Test
     fun testValidateCredentials() {
         // Valid email, password, and checkbox checked
-        assertTrue(credentialsManager.validateCredentials("example@test.com", "password123", true))
+        assertTrue(credentialsManager.validateCredentials("example@test.com", "password123"))
 
         // Invalid email
-        assertFalse(credentialsManager.validateCredentials("invalid-email", "password123", true))
+        assertFalse(credentialsManager.validateCredentials("invalid-email", "password123"))
 
         // Invalid password
-        assertFalse(credentialsManager.validateCredentials("example@test.com", "short", true))
+        assertFalse(credentialsManager.validateCredentials("example@test.com", "short"))
 
         // Checkbox not checked
         assertFalse(
```

Comment:
If you removed the checkboxChecked check - that seems obsolete
Either the comment is misleading or that assertion is obsolete